### DISCUSSION TITLE
Organizations / projects using Angular

### DISCUSSION THREAD

**Discussion by lalithpolepeddi** [madewithangular/madewithangular.github.io#432] [Show and tell]
If you're using Angular in your organization or have created an Angular project, we'd love to see it! Please share the URL with us here. 

Our goal is to feature high-profile Angular apps from public companies or venture-backed startups on [madewithangular.com/sites/](https://www.madewithangular.com/sites/), showcasing a great representation of what's possible with Angular. If your app fits this bill, it might be featured there. If not, it will remain visible here on the Made with Angular community page for everyone to see. 

Help us inspire the Angular community by contributing to this curated collection. Thanks for your support and involvement!

**Comment by sohail-ansari02** [comment]
Hello,
I created this 3D mug configurator named is "custom cup"

link: https://customcup.netlify.app/#/design

my github: https://github.com/sohail-ansari02
linkedin: https://www.linkedin.com/in/sohail-ansari-j2002

thanks

**Reply by lalithpolepeddi** [reply]
Really cool! Especially the 3D rendering and lighting - how did you go about implementing it?

**Reply by sohail-ansari02** [reply]
I used three.js for 3d rendering, for lighting setup I used HDRI maps for scene environment.

**implementation**: the basic logic is, I am getting the customised design input as user selects an design then convert that image into texture for material of model. and then apply it on the mesh.

can you please tell me when will be my project will be on madewithangular ?
 I am so excited 😊

**Reply by lalithpolepeddi** [reply]
This discussion is the Made with Angular community page, so this is where everyone will be able to see your website!

**Comment by tobintom** [comment]
Hi,
I created a project with Angular, containerized with Docker, using Spring Boot services, Eureka, Zuul gateway:
The Digital Word [https://thedigitalword.org](https://thedigitalword.org)

**Comment by awesome-yasin** [comment]
There is one popular cloud storage Indian app that uses angular its said to be Indian rival of Dropbox and Google Drive named Digiboxx.
 Check the webapp: https://app.digiboxx.com/

**Comment by lugovsky** [comment]
[UI Bakery](https://uibakery.io) is a low-code platform and its frontend is built entirely with Angular. It could be a great addition to the catalog since from the technological perspective it's really big and complex.

**Comment by PrashantSinghGour** [comment]
[IRCTC Website](https://www.irctc.co.in/) (Indian Railways Ticket booking portal) uses Angular 9.1.13.

**Reply by lalithpolepeddi** [reply]
Thanks @PrashantSinghGour! Added to the showcase [here](https://www.madewithangular.com/sites/#Indian%20Railway%20Catering%20and%20Tourism%20Corporation).

**Comment by Rajeevkumar69** [comment]
Repositree.io prides itself on its comprehensive adoption of Angular as the primary framework for all our project developments. From Angular 14 through to Angular 17, our dedicated team harnesses the versatility and power of Angular to create innovative solutions tailored to our clients' needs. By leveraging the advanced features and robust ecosystem Angular provides, we ensure that our projects are scalable, high-performing, maintainable, and future-proof. This strategic commitment to Angular underscores our dedication to delivering top-tier solutions that drive success for our organization and clients. Explore our Angular-driven projects and discover more about Repositree.io by visiting our website at https://repositree.io/

![repositree](https://github.com/madewithangular/madewithangular.github.io/assets/96237505/e925d0ca-7d48-4521-bc2c-d7fad03c7220)

**Comment by sujeetjaiswara** [comment]
Built with Angular 13
https://anywhereapp.io
https://www.oowlify.com

**Reply by sujeetjaiswara** [reply]
@lalithpolepeddi kindly check this 👆

**Comment by pavankjadda** [comment]
Aha (Video Streaming): https://www.aha.video uses Angular 16.1.1
Citi Bank (Financial): https://www.citi.com/ uses Angular 15.2.2

**Comment by nartc** [comment]
Otter AI (AI Meeting note): https://otter.ai/

**Reply by lalithpolepeddi** [reply]
Thanks @nartc! Added to the showcase [here](https://www.madewithangular.com/sites/#Otter.ai).

**Comment by BojanKogoj** [comment]
GenePlanet https://geneplanet.com

**Reply by lalithpolepeddi** [reply]
Thanks @BojanKogoj! Added to the showcase [here](https://www.madewithangular.com/sites/#GenePlanet).

**Comment by thatsamsonkid** [comment]
Amtrak (U.S Passenger Rail): https://www.amtrak.com/

**Reply by lalithpolepeddi** [reply]
Thanks @thatsamsonkid! Added to the showcase [here](https://www.madewithangular.com/sites/#Amtrak).

**Comment by gabboman** [comment]
Hello, I have created a social network that connects to the fediverse (mastodon, threads and other ones).
I created [wafrn](https://app.wafrn.net/dashboard/exploreLocal), a federated opensource social network that you could host your own node.

The frontend is [here](https://github.com/gabboman/wafrn). It uses the latest version of angular and material. The backend is [here](https://github.com/gabboman/wafrn-backend) too

**Comment by siarheihuzarevich** [comment]
<img width="2394" height="1104" alt="image" src="https://github.com/user-attachments/assets/4b9a3e54-db62-4c01-9cc4-f10b29eae43f" />

Foblex – Angular-based library and portal for building interactive node-based diagrams.
https://flow.foblex.com/

**Comment by jasonc624** [comment]
I've made a pretty big car rental management platform. Currently Angular 17.3.6. 

[SSR Landing](https://www.fumes.app)
[Owner Dashboard](https://owner.fumes.app)

Edit: Updating everything to 18+ @lalithpolepeddi

**Comment by angelokezimana** [comment]
https://irembo.gov.rw/ is using Angular (Irembo is an eGovernment platform which enables the access and provision of government services in Rwanda)

**Reply by lalithpolepeddi** [reply]
Thanks @angelokezimana! Added to the showcase [here](https://www.madewithangular.com/sites#IremboGov).

**Comment by scottstraughan** [comment]
Hello!

I have recently migrated a website for sycl.tech from Jekyll to use the Angular framework. The project currently targets Angular 18 and heavily uses Signals. It features multiple pages, filtering, searching, interactive playgrounds etc.

**The repository is available at:**
<https://github.com/codeplaysoftware/sycl.tech-website>

**And the website can be seen at:**
<https://sycl.tech>

Thank you!

**Reply by lalithpolepeddi** [reply]
Thanks @scottstraughan! Added to the showcase [here](https://www.madewithangular.com/sites#SYCL).

**Comment by Ijee** [comment]
I created an algorithm visualizer that can create and solve mazes called Amazeing.

Live site: https://amazeing.app/
Repository link: https://github.com/Ijee/Amazeing

I made it with Angular (currently v18), Bulma css, Fontawesome and a few others. Also MIT licensed.

**Comment by rBull** [comment]
Hi,

Enterprise sales CRM with AI features is made with angular - Pipeliner CRM

https://www.pipelinersales.com
https://crm.pipelinersales.com

Rado
Head of Pipeliner CRM development

**Comment by rizaasset** [comment]
[https://www.rizaasset.com/](https://www.rizaasset.com/)

**Riza Asset** is a powerful and high-performance asset management platform built with **Angular 18.2.2**, taking advantage of Firebase for real-time data management and secure hosting. We leverage the latest Angular technologies and performance optimization best practices to ensure the platform is fast, scalable, and efficient. Key highlights include:

**Standalone Components**: Our codebase is highly modular and maintainable, utilizing Angular’s new standalone component architecture. This reduces dependency on NgModules and promotes an efficient build process, making the app faster and more scalable.

**Angular Signals**: We’ve integrated Signals for highly responsive, reactive programming, ensuring our data flows are smooth and consistent across different components, improving performance and user experience.

Riza Asset’s robust asset management capabilities are perfect for enterprises looking for real-time collaboration, asset tracking, and detailed reporting, all in a highly interactive and modern web interface.

Explore our site and experience the future of asset management technology built with Angular!
Visit us @ [https://www.rizaasset.com/](https://www.rizaasset.com/)

**Comment by Karakaz** [comment]
Made a tool for planning playthroughs of Dragon Age: Origins in regards to which abilities to choose for one's characters. Components stays true to original art direction and are custom made for the most part. Desktop only support for now.

Check it out https://tevinter.net/dao/planner

**Comment by scottstraughan** [comment]
Hello again!

I've made another small tool that can check the OpenSSF Scorecards for an entire user/organization on GitHub/GitLab/others. It features caching, theming (dark-mode etc), searching, filtering, ordering etc.

**The tool is available at:**
<https://www.scoredash.xyz>

**The source code is available at:**
https://github.com/scottstraughan/openssf-scorecard-dashboard

Thanks again for the excellent website!

**Comment by m2a2x** [comment]
Hello

Wanted to share [DataGrid for Angular](https://github.com/revolist/angular-datagrid) a powerful and flexible component.

⚡ Features:
	•	High performance for large datasets
	•	Fully customizable and lightweight
	•	Two-way data binding
	•	Virtual scrolling
	•	Rich API for advanced use cases


https://github.com/revolist/angular-datagrid

**Comment by sohail-ansari02** [comment]
Hi
I recently made an website for catering services , used angular SSG
https://kgncaterers.in

**Comment by juanitourquiza** [comment]
Hi,

I made this website:
https://declaracion.abgalapagos.gob.ec/

**Comment by EzequiasAlmeida** [comment]
Company developer here. Our entire platform is made using Angular
https://my.adoreal.com/
More about the business here: https://www.adoreal.com/en

**Comment by michael-small** [comment]
H&R Block's internal site "MyBlock" for filing. Without having an account, you can't really see it unless you click on the "Sign in to MyBlock" login page, and that is quite limited. But it is a very nice site post-login too lol. 

https://login.taxes.hrblock.com/hrblock/login/LoginRegistration.hrbx?TaxType=TCF

**Reply by lalithpolepeddi** [reply]
Nice find @michael-small! Added to the showcase [here](https://www.madewithangular.com/sites/#MyBlock%20-%20H&R%20Block).

**Comment by michael-small** [comment]
Google Cloud Console

https://console.cloud.google.com/

**Reply by lalithpolepeddi** [reply]
Thanks @michael-small! Added to the showcase [here](https://www.madewithangular.com/sites#Google%20Cloud%20console).

**Comment by rBull** [comment]
Hi,

did you have a chance to look at this one?
https://github.com/madewithangular/madewithangular.github.io/discussions/432#discussioncomment-10214672


Enterprise sales CRM with AI features is made with angular - Pipeliner CRM

https://www.pipelinersales.com/
https://crm.pipelinersales.com/

Rado
Head of Pipeliner CRM development

**Reply by lalithpolepeddi** [reply]
Yep, thanks for sharing again! The main showcase features a curated selection of projects, and this discussion is a great space for listing all kinds of projects made with Angular. Always inspiring to see what people are creating with Angular, thanks for sharing!

**Comment by michael-small** [comment]
Bitwarden client

Repo: https://github.com/bitwarden/clients/tree/main

Page using it: https://vault.bitwarden.com/#/login (maybe there is a better page w/o a login but I am not really a Bitwarden user, learned this from [this thread](https://old.reddit.com/r/angular/comments/1iyv832/where_can_i_find_good_github_repos_for_decent/))

**Reply by lalithpolepeddi** [reply]
Nice one @michael-small! Added to the showcase [here](https://www.madewithangular.com/sites#Bitwarden%20Vault).

**Comment by lawaldare** [comment]
Company Frontend Developer here. Our entire pages (Ligand, Entry, Complex, Domain, Protein) are made using Angular

link: https://www.ebi.ac.uk/pdbe-srv/pdbechem/chemicalCompound/show/sti (v18.2.1) - This is our new Ligand Page.

All our pages are based with Angular and I am building and managing them. 

More about the company: https://www.ebi.ac.uk/about/

**Reply by lalithpolepeddi** [reply]
Great work @lawaldare! I'm a big fan of EMBL-EBI. Added to the showcase [here](https://www.madewithangular.com/sites#European%20Bioinformatics%20Institute%20(EMBL-EBI)).

**Reply by lawaldare** [reply]
Thanks @lalithpolepeddi

**Comment by MansurIsakov** [comment]
Hey, uzbek fintech product https://payme.uz/ made using Angular 🇺🇿

**Reply by lalithpolepeddi** [reply]
Thanks @MansurIsakov! Added to the showcase [here](https://www.madewithangular.com/sites#Payme).

**Comment by michael-small** [comment]
Another site behind a PW/UN lol, but https://app.bluecareadvisormn.com/#/app/home is a 17.3.12 site

**Reply by michael-small** [reply]
I came here to see if I had already posted it - they are up to 19.2.14 woo

**Comment by devidevio** [comment]
Landing Page:
* [https://papersend.io/](https://papersend.io/) built with [Astro](https://astro.build/).

Application Page: 
* [https://app.papersend.io/](https://app.papersend.io/) built with Angular 19.1

Tech Stack:
- [Angular 19.1](https://angular.dev/) as my Framework of trust.
- [PrimeNG](https://primeng.org/) as the Component Library.
- [Supabase](https://supabase.com/) for the Backend.
- [TailwindCSS](https://tailwindcss.com/) for Styling.
- [Cloudflare Pages](https://pages.cloudflare.com/) for Deployment.

Built with focus on:
* [Signals](https://angular.dev/api/core/signal)
* [Effects](https://angular.dev/api/core/effect)
* Built-in [control flow](https://angular.dev/guide/templates/control-flow)
* [afterRender](https://angular.dev/api/core/afterRender) / [afterNextRender](https://angular.dev/api/core/afterNextRender) lifecycle hooks
* Completely [Zoneless Change Detection](https://angular.dev/guide/experimental/zoneless)
* [View Transitions](https://angular.dev/api/router/withViewTransitions) out of the box
* Function Based Routing Guards
* New [@Input](https://angular.dev/guide/components/inputs) and [@Output](https://angular.dev/guide/components/outputs) decorators

Some of the features I heavily rely on are still in developer or even experimental state, but they work already awesome for my cases.

**Reply by lalithpolepeddi** [reply]
Looks great @devidevio! Thanks for taking the time to share the stack and Angular features - really appreciate the context and links.

**Reply by devidevio** [reply]
Appreciate your feedback, thanks a lot :)

**Reply by sujeetjaiswara** [reply]
🌟 Fantastic implementation of Angular 19.1 with PrimeNG and Supabase! The use of TailwindCSS for styling and Cloudflare Pages for deployment showcases a robust and modern tech stack. Your focus on signals, effects, and zoneless change detection is impressive. Great job leveraging experimental features effectively! 🚀

**Comment by SkyZeroZx** [comment]
Create an ecommerce with Angular | SSR | ISR (Redis Cache) | PWA with all the possible optimization techniques lazy load by components, own design system, SEO, switching in runtime mobile or desktop view based on the REQUEST DI Token user agent, possibility of packaging to upload the play store as APK

Tech Stack:
[Angular 19.2](https://angular.dev/) FrontEnd Framework
[TailwindCSS](https://tailwindcss.com/)  Styling
[VPS Contabo](https://contabo.com/en/) Deployment
[Docker](https://www.docker.com/) Container
[Redis](https://redis.io/es/) Cache
[Redis](https://nestjs.com/) NestJS

https://kikesport.com.pe/

**Reply by sujeetjaiswara** [reply]
Impressive work on integrating Angular 19.2 ❤️ with SSR, ISR, and PWA capabilities! The use of Redis Cache and NestJS 💜 for optimization, along with TailwindCSS for styling, demonstrates a well-thought-out approach. Packaging for Play Store deployment is a great addition. Kudos on a comprehensive and efficient tech stack!

**Reply by lalithpolepeddi** [reply]
Nice @SkyZeroZx! Thanks for sharing your stack.

**Comment by michael-small** [comment]
https://angular.love/

edit: if any particular page other than the TLD is posted, I think this page on why the blog finally was rewritten in Angular would be a great thing to point at: https://angular.love/why-we-migrated-our-blog-from-wordpress-to-angular

Very meta and a great site overall

**Comment by scottstraughan** [comment]
https://www.saorsail.com

Angular based front-end that uses the F-Droid database. Usual stuff included such as searching, filtering, categorisation, pagination, theming etc. Also heavily uses IndexDB/Local Storage.

I also wrote an Android companion app for this allowing users to "push to install" directly to their devices.

Development is still ongoing (in my spare time), happy for anyone else to help!

Cheers!

**Comment by ilhanyumer** [comment]
https://dskhome.bg

All pages are made with Angular. It is a very popular internet real estate web site in Bulgaria.

**Comment by bogusred** [comment]
<img width="1408" height="1152" alt="image" src="https://github.com/user-attachments/assets/32317675-c18f-4a31-b24a-ecbf10a231f8" />

https://www.paperdemon.com/

I've made PaperDemon Art RPG with Angular! 

PaperDemon is an online art roleplaying game providing community and inspiration for artists around the world. Artists have created over 7000 characters in our art game. We have over 300 monthly active players (logged in).

PaperDemon is a bootstrapped startup, not VC funded. Thank you for considering including us in your showcase and have a lovely day!

**Comment by hisham** [comment]
Essential Clinic - https://www.essentialclinic.ca/

One of Canada's top online clinics. Angular 20.3.15 + Scully for SSG.

**Comment by RUGWIRO-Olivier** [comment]
**Project Name:** Resumemind
**URL:** https://resumemind.com

**Description:**
I built Resumemind to solve the common problem of generating clean, printable PDFs from the web without using canvas or heavy libraries. It is a free, privacy-focused resume builder.

**Key Angular Features Used:**
* **Dynamic Component Rendering:** To swap between different resume templates instantly.
* **Reactive Forms:** Handling complex user input for work history and skills.
* **Scoped Styles:** Managing the strict CSS `@media print` rules to ensure page breaks don't cut text in half.

I'd love to share how we handled the print-media challenges if anyone is interested!

**Comment by smartredfox** [comment]
Online Banking for NatWest (the uk’s biggest high street bank) runs on Angular. They use the same system to also run online banking for other brands in the NatWest group (RBS, Isle of Man bank, Ulster bank, and NatWest International). 

[Online Banking](onlinebanking.natwest.coml)

**Reply by lalithpolepeddi** [reply]
Nice find @smartredfox! Added to the showcase [here](https://www.madewithangular.com/sites/#NatWest%20Bank).

**Comment by mainandmakers** [comment]
Hi, I'm Kevin. I developed this **Angular Web App** to run alongside the Website as part of my overall **Main and Makers** project. Main and Makers is a web space for Makers, the Artisans of Greater Main Street. The Web App includes **Posts, Games, and more**.

**Angular Web App:** https://app.mainandmakers.com  
**Website:** https://mainandmakers.com

---

## Tech Stack
- Angular 21 (Current Version)
- GSAP (Animation)
- Phaser (Game Engine)
- Custom CSS
- HTML5 Video API
- Web Audio API
- WordPress REST API (Backend)

---

## Angular‑Specifics
- Standalone Components
- Services
- Zoneless
- Signals
- Effects
- Computed
- RxJS
- New Control Flow Syntax

---

## Creative Tools
- Canva

---

<img width="1920" height="1080" alt="MainAndMakers" src="https://github.com/user-attachments/assets/14c2b06b-cd4f-4143-abea-3eb72bc004c8" />

**Comment by APUNCode** [comment]
[Spin the Wheel](https://spinthewheel.io), a highly configurable random selection app used worldwide runs on Angular. It has over 1 million registered users, mainly teachers who use it in the classroom in a variety of educational ways.

It might look simple as first but look at the incredible functionality:

**Key features:**
Multiple wheel spinners on one page:
Build pages with [up to eight wheels](https://spinthewheel.io/article/create-multiple-random-decision-wheels) and spin them individually or together.

Custom text, palettes and themes:
Change segment labels, add images to the background or on the wheel itself and use [themes](https://spinthewheel.io/more/classroom-name-picker-50-themes) for events and holidays.

Sounds and animations:
Add sound effects and post-spin animation effects to make spins more exciting in classrooms, meetings and streams.

If/then rules (Post-spin reaction effects):
Decide what happens after each spin, with [different reactions depending](https://spinthewheel.io/article/post-spin-reactions-video) on where the wheel stops.

Weighting of segments:
Alter the probability of individual wheel segments by using weighting functionality.

Provably fair spins:
Use this for giveaways, competitions, prize draws, classrooms, events, or any spin where [proof of fairness](https://spinthewheel.io/more/provably-fair-spins) matters. Verified replays show the original result and timestamp.

Music entries:
Use this for lessons, sight-reading, warmups, rehearsals, classroom games, or practice activities where each spin reveals a notated piece of [playable sheet music](https://spinthewheel.io/more/musicxml-sheet-music), powered by MusicXML.

Save your wheels:
Create an account to save wheels, duplicate and tweak them.

Use Spin the Wheel instantly on spinthewheel.io, or install the iOS and Android apps for a faster, more native experience.

We also maintain a [Spin the Wheel Discord bot](https://spinthewheel.io/more/discord-bot) for communities that want to run random picks, games and decisions directly inside Discord.

**Comment by filozopdasha** [comment]
Hey! Our team has been working on a pivot table library that will perfectly suit Angular apps.

---

## [Flexmonster Pivot Table & Charts component](https://www.flexmonster.com/demos/angular/pivot-table/)

**GitHub:** https://github.com/flexmonster/ng-flexmonster 

Flexmonster is a feature-rich Flexmonster Pivo web component for embedding interactive data analytics and reporting into any Angular application. It's an enterprise-grade pivot grid library for visualizing and exploring complex datasets in the browser.

## Key pivot grid features:

- **Pivot table & charts** -group, filter, sort, and drill through data, then visualize it interactively
- **High performance on large data** - handles 1M+ rows of grouped data and datasets over 1GB with fast rendering and smooth scrolling
- **Fully customizable** - match your app's design, configure layouts (compact/classic/flat), conditional and number formatting, and an extensive JavaScript API to build any reporting scenario
- **Connects to any data source** - JSON, CSV, SQL, and NoSQL databases, OLAP cubes, Elasticsearch, MongoDB, or custom sources
- **Export & share** - export reports to PDF, Excel, HTML, CSV, or PNG, or share via link
- **Localization & accessibility** - built-in i18n and accessibility support

**Angular integration:** to get started quickly, use [the official Angular wrapper](https://www.flexmonster.com/doc/integration-with-angular/).


**Angular pivot table integration tutorial:** https://www.youtube.com/watch?v=hbaVewUYZRE&t=7s

**Comment by ajitzero** [comment]
Epic Games uses Angular (21.2.5) for part of its developer documentation website: https://dev.epicgames.com/documentation/

I remember it being entirely Angular a few years ago, but the landing page seems to be Next.js now. Please add the exact URL above.

Edit: I have no affiliation with Epic Games Store. Just sharing it here.

**Reply by lalithpolepeddi** [reply]
Thanks @ajitzero! Added to the showcase [here](https://www.madewithangular.com/sites#Epic%20Games%20Product%20Documentation).

**Comment by PascaleBeier** [comment]
HitKeep uses angular 22 as its frontend framework:

Repository: https://github.com/PascaleBeier/hitkeep

Website: https://hitkeep.com

<img width="1495" height="838" alt="image" src="https://github.com/user-attachments/assets/6f187ff2-839f-45ba-9836-57e4db191628" />

**Reply by lalithpolepeddi** [reply]
I was looking for an analytics solution like this, thanks for sharing @PascaleBeier!

**Reply by PascaleBeier** [reply]
Verrry glad to hear! Let me know if there's a feature you'd love 🙂

**Comment by aabitbolasp-art** [comment]
MinuteShark uses Angular - a time tracker with an AI assistant that actually does stuff for you (not just a chatbot): log time, get reports etc.

Link [https://minuteshark.com/](https://minuteshark.com/)

<img width="2018" height="1464" alt="ai-app-screenshot-mockuponly" src="https://github.com/user-attachments/assets/8e7dd9c2-7945-4c3c-9582-02aab13a213c" />

**Comment by CobyBoy** [comment]
https://www.despegar.com.ar/ 
Argentine unicorn branch of one of the largest online travel agencies (OTA) in Latin America. 
The company allows users to search, compare, and book a wide variety of travel services through its website and mobile app.

**Reply by lalithpolepeddi** [reply]
Thanks @CobyBoy! Added to the showcase [here](https://www.madewithangular.com/sites#Despegar).

**Comment by CobyBoy** [comment]
Here are some more websites using Angular that could be added to the list.

## 🇦🇷 Argentina

### Financial Services / Banking

* **Naranja X** — https://www.naranjax.com/
  Argentine financial services platform/fintech, originally evolved from Tarjeta Naranja.

* **Banco Supervielle** — https://personas.supervielle.com.ar/obi/usuarios/login
  Online Banking portal for the Argentine bank Supervielle.
  Note: the main website (https://www.supervielle.com.ar/personas) does not use Angular, but the Online Banking application does.

* **Banco Ciudad** — https://www.bancociudad.com.ar/institucional/
  Argentine public bank and financial agent of the City of Buenos Aires (CABA).

* **ICBC Argentina** — https://onlinebanking.icbc.com.ar/shell/#/login
  Online Banking portal for ICBC Argentina.

### Insurance

* **Mercantil Andina** — Argentine insurance company.
  The main website (https://mercantilandina.com.ar/) does not use Angular, but several of its applications do:

  * Third-party claims: https://siniestros.mercantilandina.com.ar/terceros/#/login
  * Car repair shop self-management platform: https://siniestros.mercantilandina.com.ar/tasacion/taller/#/login
  * Insurance producers/advisors portal: https://servicios.mercantilandina.com.ar/sigmav3/
  * Policyholder portal: https://asegurados.mercantilandina.com.ar/#/login

* **La Segunda Seguros** — https://www.lasegunda.com.ar/
  Argentine insurance company/group.

* **Grupo San Cristóbal** — https://www.gruposancristobal.com.ar/
  Argentine insurance and financial services group originally founded in Rosario.

### Technology / Payments

* **Prisma Medios de Pago – Developer Portal** — https://developers.prismamediosdepago.com/portal/api/catalog
  Developer/API portal for Prisma Medios de Pago, an Argentine payments technology and processing company.

* **NOVIX** — https://novix.com/
  Technology and business consulting company focused on data analytics, business planning, and customized technology solutions across Latin America.

### Government

* **Poder Judicial de la Nación (PJN)** — https://pjn.gov.ar/
  Official website of Argentina's federal/national judiciary, providing access to court cases, proceedings, directories, rulings, forms, and other judicial services.

### Other Industries

* **RODE** — https://rode.com.ar/
  Argentine construction and real estate development company based in Córdoba.

* **BEX Patagonia (Baños Exclusivos de la Patagonia)** — https://bexpatagonia.com.ar/
  Argentine company specializing in portable sanitation solutions and mobile bathrooms.

* **Álvarez Filtros / AR Distribuidora** — https://www.alvarezfiltros.com.ar/
  Argentine distributor of automotive, agricultural, and industrial filters based in General Baldissera, Córdoba.

## 🌎 Other Countries / International

* **PepsiCo Jobs** — https://www.pepsicojobs.com/main/
  Official careers and recruitment website for PepsiCo.

* **Banco de Bogotá – Virtual Banking** — https://virtual.bancodebogota.co/
  Online Banking portal for Banco de Bogotá, Colombia.

* **Citi** — https://www.citi.com/
  Banking and financial services website for Citi.

* **Bloom & Wild** — https://www.bloomandwild.com/
  UK-based online flower delivery and gifting company, known for its letterbox flowers and flower subscriptions.

**Reply by lalithpolepeddi** [reply]
Thank you @CobyBoy for taking the time to find and send all these sites! I've added the "Financial Services / Banking" and "Other Countries / International" sites to the [showcase](https://www.madewithangular.com/sites); the rest are here for everyone to see. Thanks again!
